src/ucp/rndv/rndv_put.c

```diff
@@ -80,6 +95,12 @@ ucp_proto_rndv_put_common_flush_completion_send_atp(uct_completion_t *uct_comp)
     const ucp_proto_rndv_put_priv_t *rpriv = req->send.proto_config->priv;
 
     ucp_trace_req(req, "rndv_put_common_completion_send_atp");
+
+    if (ucs_unlikely(uct_comp->status != UCS_OK)) {
```

Comment:
minor: maybe also add status to the trace above?

Reply:
Done.